### DISCUSSION TITLE
Width Percentage

### DISCUSSION THREAD

**Discussion by Hacki01** [barvian/fluid-tailwind#67] [Help]
Hello! 

I'm new here and I have a quick question, is it possible to use percentages to change width?
I couldn't find an answer anywhere.
Something like that:
`~w-[70%]/[95%]`
The error is:
`/* error - Start value `70%` is not a length */ {}`

**Comment by barvian** [comment]
Hi 👋, not right now unfortunately. Only lengths (e.g. 20px, 2rem) work due to CSS limitations.
